app/plugins/monitoring/monitoring.css

```diff
@@ -1,21 +1,3 @@
-/*
- * This file is part of a module with proprietary Enterprise Features.
- *
- * Licensed to Crate.io Inc. ("Crate.io") under one or more contributor 
- * license agreements.  See the NOTICE file distributed with this work for
- * additional information regarding copyright ownership.
- *
- * Unauthorized copying of this file, via any medium is strictly prohibited.
- *
- * To use this file, Crate.io must have given you permission to enable and
- * use such Enterprise Features and you must have a valid Enterprise or
- * Subscription Agreement with Crate.io.  If you enable or use the Enterprise
- * Features, you represent and warrant that you have a valid Enterprise or
- * Subscription Agreement with Crate.io.  Your use of the Enterprise Features
- * if governed by the terms and conditions of your Enterprise or Subscription
- * Agreement with Crate.io.
- */
-
```

Comment:
I think we should add the Crate APL header here instead.

Reply:
That also crossed my mind. I will just copy in the file header from the main CrateDB sources like [1], right?

[1] https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/planner/consumer/InsertFromSubQueryPlanner.java

Reply:
> I think we should add the Crate APL header here instead.

All other files also lack respective headers. Shall we take care of that in a followup PR in order to keep the scope of this to semantic changes and the other patch for filling the gaps?

Reply:
Ah all other files don't have any header, sorry wasn't aware of this. So ok lets just remove it for now as otherwise we should add a license header in general for everything dedicated. Not sure if that is needed..